build.gradle

```diff
@@ -0,0 +1,138 @@
+//////////////////////////////
+// Buildscript Dependencies //
+//////////////////////////////
+buildscript {
+    repositories {
+        jcenter()
+        maven { url 'https://palantir.bintray.com/releases/' }
+    }
+
+    dependencies {
+        //classpath 'com.palantir.gradle.revapi:gradle-revapi:1.2.0'
+        classpath 'com.netflix.nebula:gradle-info-plugin:7.1.2'
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
+        classpath 'com.netflix.nebula:nebula-publishing-plugin:17.0.5'
+    }
+}
+
+/////////////
+// Plugins //
+/////////////
+apply plugin: 'java'
+apply plugin: 'idea'
+
+// intentionally not applying nebula.maven-publish, but most of its constituent
+// plugins, because we do _not_ want nebula.maven-compile-only
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+apply plugin: 'nebula.maven-nebula-publish'
+apply plugin: 'nebula.maven-base-publish'
+apply plugin: 'nebula.maven-developer'
+apply plugin: 'nebula.maven-manifest'
+apply plugin: 'nebula.info-scm'
+apply plugin: 'nebula.maven-scm'
+apply plugin: 'nebula.javadoc-jar'
+apply plugin: 'nebula.source-jar'
+
+/////////////////////////
+// Module Dependencies //
+/////////////////////////
+repositories {
+    jcenter()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+}
+
+// keep group/version info the same as from before
+group 'com.palantir.patches.sourceforge'
+version '3.0.3-p6'
```

Comment:
Intentionally hardcoded for now. Happy to swap over to git-version plugin, but would require doing some tagging and better git management.